lib/charms/operator_libs_linux/v1/snap.py

```diff
@@ -976,19 +976,27 @@ def _system_set(config_item: str, value: str) -> None:
         raise SnapError("Failed setting system config '{}' to '{}'".format(config_item, value))
 
 
-def hold_refresh(days: int = 90) -> bool:
+def hold_refresh(days: int = 90, forever: bool = False) -> bool:
```

Comment:
**suggestion** I generally prefer to have one argument if things can be conflicting, like 'if I have hold forever true, what's the point of providing the days'. 

This is to say that I'd rather clean the signature like 

days: optional [int]

If days is none, we hold forever. The default can still be 90 though.

Anyhow, it is not a super big point, hence the suggestion prefix. But if other reviewers feel the same way, it may be worth addressing

Reply:
Conversation already happened here: https://github.com/canonical/operator-libs-linux/issues/80